src/util.ts

```diff
@@ -288,13 +288,17 @@ export function accessPathWithDatum(path: string, datum = 'datum') {
  * @param datum The string to use for `datum`.
  */
 export function flatAccessWithDatum(path: string, datum: 'datum' | 'parent' | 'datum.datum' = 'datum') {
-  return `${datum}[${stringValue(splitAccessPath(path).join('.'))}]`;
+  return `${datum}[${doubleQuotes2Single(stringValue(splitAccessPath(path).join('.')))}]`;
 }
 
 function escapePathAccess(string: string) {
   return string.replace(/(\[|\]|\.|'|")/g, '\\$1');
 }
 
+function doubleQuotes2Single(string: string) {
```

Comment:
No, we can't do this. It's too brittle and possibly introduces errors. If we want to change the quotes, we need to extend `stringValue` to accept a quote parameter.